### DISCUSSION TITLE
我是在windows2012环境下，不管怎么搞都不能使用nonobot的包

### DISCUSSION THREAD

**Discussion by xia2233** [nonebot/discussions#137] [问答]
![屏幕截图 2022-05-02 230414](https://user-images.githubusercontent.com/104424500/166257536-cc1d4b48-e6b7-442f-9cfa-1d6074d5bf84.jpg)
![屏幕截图 2022-05-02 230447](https://user-images.githubusercontent.com/104424500/166257540-fdb91fea-5cba-42c0-86d6-3eb9af564b78.jpg)
是缺少什么东西吗？

**Comment by yanyongyu** [comment]
没有安装nonebot2和nonebot-adapter-onebot
